realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -599,6 +601,10 @@ void BaseRealSenseNode::setupStreams()
                         }
                         else
                         {
+                            f = disparity_in->process(f);
```

Comment:
We may want to consider adding a parameter to enable/disable filtering.  We can probably address this with the upstream team.  I seem to remember this being in there...feels like other things are missing.